packages/core/strapi/src/node/create-build-context.ts

```diff
@@ -135,10 +135,11 @@ const createBuildContext = async <TOptions extends BaseOptions>({
   const features = strapiInstance.config.get('features', undefined);
 
   const { bundler = 'vite', ...restOptions } = options;
+  const serveAdminPanel = strapiInstance.config.get('admin.serveAdminPanel', true);
 
   const buildContext = {
     appDir,
-    basePath: `${adminPath}/`,
+    basePath: serveAdminPanel ? `${adminPath}/` : '/',
```

Comment:
Should this be the same as your v4 version so users can set the admin.url to use?

Reply:
I completly change the way this is created in my reverse proxy PR can you check if the issue is already fixed there instead ?

Reply:
I won't port my PR to v4 so you can def go ahead. for this one it's fine too, I was more concerned that my PR was breaking the fix. you can merge it if it fixes what it says it does 💯  I don't know when my fix will be merged we need more testers

Reply:
@alexandrebodin I will test your PR throughly along with mine tomorrow!